e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -122,6 +125,11 @@ export const navigateToEntitySelectionStep = (
     entityPickerModal().within(() => {
       cy.findByText(resourceType).click();
       cy.findAllByText(resourceName).first().click();
+
+      // Collection picker requires an explicit confirmation.
+      if (experience === "browser") {
+        cy.findByText("Select").click();
+      }
```

Comment:
We show an explicit Select button for collection browsers. Just so it's easier to navigate to the nested collections.

<img width="1566" height="1406" alt="CleanShot 2568-08-20 at 04 24 33@2x" src="https://github.com/user-attachments/assets/c711336f-1a24-4960-89fe-f3fd196c406e" />